models/DataObject/Localizedfield/Dao.php

```diff
@@ -150,7 +150,11 @@ public function save($params = [])
                     ];
 
                     if ($fd instanceof DataObject\ClassDefinition\Data\Relations\AbstractRelations) {
-                        if ($this->model->isLanguageDirty($language) || $params['saveLocalizedRelations']) {
+                        if ( ($params['saveRelationalData']['saveLocalizedRelations']
```

Comment:
I guess we need an `isset($params['saveRelationalData'])` here as well, in order to avoid notice log messages.